scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -165,72 +168,62 @@ sub load_all_variation_sets {
   my $sql = qq{INSERT INTO variation_set_variation (variation_id, variation_set_id ) VALUES (?, ?)};
   my $sth = $dbhvar->prepare($sql);
 
-  open FH, "<", "$load_file" or die "Can not open $load_file: $!";
-  while (<FH>) {
+  open my $load_vs, "<", "$load_file" or die "Can not open $load_file: $!";
+  while (<$load_vs>) {
     chomp;
     my $var_id = (split)[0];
     my $var_set_id = (split)[1];
     
     $sth->execute($var_id, $var_set_id);
```

Comment:
I doubt this can happen as this is only done after a new dbsnp import. In the past i had the constraint `variation_set_id = ''` but had to change that because of the rerun.